includes/definitions/fs-net-pdu.yaml

```diff
@@ -0,0 +1,13 @@
+os: fs-net-pdu
+text: 'FS.COM monitored pdu'
+type: power
+icon: fs
+group: fs
+over:
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_current, text: Current }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1
```

Comment:
Please use the complete OID here, cause .1.3.6.1.4.1 is far to wide. It covers all devices of all brands ... 

You have to use .1.3.6.1.4.1.30966 which restricts to the company that makes these PDUs ...

Reply:
Hi, sorry for the delay, I did the changes, I hope it's ok.

Thanks a lot-